src/components/auth/Register.js

```diff
@@ -0,0 +1,169 @@
+            });
+        })
+        .catch(err => {
+          console.log("Registration Error", err);
+          this.setState({ errorStatus: true });
+        });
+    }
+  }
+
+  render() {
+    const { user, submitted } = this.state;
+    return (
+      <div className="container">
+        <div className="boxLeft">
+          <img className="logo" src={GroaWhite} alt="groa logo" />
+        </div>
+        <div className="boxRight">
+          <form className="form" onSubmit={this.handleSubmit}>
+            <h2>Register</h2>
+            {/* divs with changing classnames updates error handling for form */}
+            <div
+              className={
+                "forms" + (submitted && !user.email ? " has-error" : "")
+              }
+            ></div>
+            <input
+              className="form-control"
+              type="text"
+              name="email"
+              value={user.email}
+              onChange={this.handleChange}
+              placeholder="Email"
+            />
+            {submitted && !user.email && (
+              <div className="callingError">Email is required</div>
+            )}
+
+            <div
+              className={
+                "forms" + (submitted && !user.user_name ? " has-error" : "")
+              }
+            ></div>
+            <input
+              className="form-control"
+              type="text"
+              name="user_name"
+              value={user.user_name}
+              onChange={this.handleChange}
+              placeholder="Username"
+            />
+            {submitted && !user.user_name && (
+              <div className="callingError">Username is required</div>
+            )}
+
+            <div
+              className={
+                "forms" + (submitted && !user.password ? " has-error" : "")
+              }
+            ></div>
+
+            <input
+              className="form-control"
+              type="password"
+              name="password"
+              value={user.password}
+              onChange={this.handleChange}
+              placeholder="Password"
+            />
+            {submitted && !user.password && (
+              <div className="callingError">Password is required</div>
+            )}
+
+            <div
+              className={
+                "forms" +
+                (submitted && !user.confirmpassword ? " has-error" : "")
+              }
+            ></div>
+            <input
+              className="confirmPass form-control"
+              type="password"
+              name="confirmpassword"
+              value={user.confirmpassword}
+              onChange={this.handleChange}
+              placeholder="Confirm Password"
+            />
+            {submitted && !user.confirmpassword && (
+              <div className="callingError">Confirm Password is required</div>
+            )}
+
+            <div className="BottomLogin">
+              <button className="LoginBtn">Login </button>
```

Comment:
I still don't know what the registering that was here was for, but it didn't serve a purpose as far as I know.

Reply:
I don't know why it was there either, and I agree, I don't think it served any purpose.